libavcodec/vaapi_encode.c

```diff
@@ -264,9 +266,11 @@ static int vaapi_encode_make_tile_slice(AVCodecContext *avctx,
 }
 
 static int vaapi_encode_issue(AVCodecContext *avctx,
-                              VAAPIEncodePicture *pic)
+                              HWBaseEncodePicture *base_pic)
```

Comment:
Don't change the argument. you may cast `VAAPIEncodePicture` pointer to `HWBaseEncodePicture` pointer in `vaapi_encode_issue`, like what you did for `vaapi_encode_wait`

Reply:
This is used for base layer to call .issue. So it should use base argument. Thanks.